i18n/en.json

```diff
@@ -3002,7 +3002,8 @@
   "convert_channel.title": "Convert {display_name} to a Private Channel?",
   "copied.message": "Copied",
   "copy_url_context_menu.getChannelLink": "Copy Link",
-  "copy.block.message": "Copy code block",
```

Comment:
`copy.block.message` is deleted because it was only used in `CopyButton`. Should I also delete it from other language files? OR deleting from `en.json` is enough?